app/controllers/auth/registrations_controller.rb

```diff
@@ -28,6 +28,8 @@ def build_resource(hash = nil)
     resource.invite_code = params[:invite_code] if resource.invite_code.blank?
     resource.agreement   = true
 
+    resource.last_sign_in_ip = request.remote_ip if resource.last_sign_in_ip.nil?
```

Comment:
I'm not sure there is any situation when `resource.last_sign_in_ip` is not nil here? Also, current_sign_in_ip is probably enough, don't see a need for last.

Reply:
Removed the assignment to `last_sign_in_ip`.